modules/ROOT/pages/redis/redis-connector-reference.adoc

```diff
@@ -3,1264 +3,1145 @@ ifndef::env-site,env-github[]
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable |  #[payload] |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | 
 |===
 
-=== Output
-
-[cols=".^50%,.^50%"]
+==== Output
+[%autowidth.spread]
 |===
-| Type a| Boolean
+|Type |Boolean
 |===
 
 === For Configurations
+* <<redis>> 
 
-* <<redis>>
-
-=== Throws
-
-* REDIS:UNKNOWN
-* REDIS:CONNECTIVITY
-* REDIS:INVALID_REQUEST_DATA
-* REDIS:UNABLE_TO_UNSUBSCRIBE
-* REDIS:INVALID_STRUCTURE_FOR_INPUT_DATA
-* REDIS:RETRY_EXHAUSTED
-* REDIS:CONNECTIVITY
+==== Throws
+* REDIS:CONNECTIVITY 
+* REDIS:INVALID_REQUEST_DATA 
+* REDIS:INVALID_STRUCTURE_FOR_INPUT_DATA 
+* REDIS:RETRY_EXHAUSTED 
+* REDIS:UNKNOWN 
+* REDIS:UNABLE_TO_UNSUBSCRIBE 
 
 
 [[pushToList]]
-== Push To List
+=== Push To List
 `<redis:push-to-list>`
 
 
-Push the message payload to the desired side (LEFT or RIGHT) of the list stored at the specified key. If key does not exist, a new key holding a list is created as long as ifExists is not true.
+Push the message payload to the desired side (LEFT or RIGHT) of the list stored at the specified key. If key does not exist, a new key holding a list is created as long as ifExists is not true. 
 
 
-=== Parameters
-
-[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+==== Parameters
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
 | Configuration | String | The name of the configuration to use. | | x
 | Key a| String |  Key to use for LPUSH/RPUSH/LPUSHX/RPUSH |  | x
-| Value a| String |  The value to push. |  #[payload] |
-| Push Left a| Boolean |  The side where to push the payload, either LEFT or RIGHT |  false |
-| If Exists a| Boolean |  If true execute LPUSHX/RPUSH otherwise LPUSH/RPUSH |  false |
-| Target Variable a| String |  The name of a variable that stores the output from the operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable |  #[payload] |
+| Value a| String |  The value to push. |  `#[payload]` | 
+| Push Left a| Boolean |  Side that receives the pushed payload. If the parameter is true, push the payload on the left side. If the parameter is false, push the payload on the right side. |  false | 
+| If Exists a| Boolean |  If the parameter is true, execute LPUSHX or RPUSH. If the parameter is false, execute LPUSH or RPUSH. |  false | 
```

Comment:
I think the note on this line is worth discussing with a dev. First, I wonder if maybe it should be "LPUSHX/RPUSHX" for true and "LPUSH/RPUSH" for false, and the dev made a tiny and extremely dangerous typo. (I do see that line 922 also has "LPUSH/RPUSH/LPUSHX/RPUSH".) Second, I think LPUSH/RPUSH means "LPUSH or RPUSH" but it might mean "LPUSH and then RPUSH". Third, references to functions like LPUSH should probably be replaced with a more English description of the effects. For example, "If the parameter is true and the list exists, push the message payload onto the list. If the parameter is true and the list does not exist, do nothing. If the parameter is false, push the message payload onto the list, creating a new list if one does not exist already." This example may or may not match the actual behavior of the function, which is what I'd like to see explained more fully by a dev.

Reply:
Checking with the devs

Reply:
Good catch. Fixed to show LPUSHX or RPUSHX. Thanks!